rosapi/src/rosapi/objectutils.py

```diff
@@ -65,6 +70,12 @@ def get_typedef(type):
          - string[] constnames
          - string[] constvalues
     get_typedef will return a typedef dict for the specified message type"""
+
+    # Check if the type string indicates a sequence (array) type
+    if type.startswith("sequence<") and type.endswith(">"):
+        # Extract the inner type and continue processing
+        type = type[9:-1]
```

Comment:
Could we make this a regex instead of relying on hard-coded numbers?

```suggestion
    if matches := re.findall("sequence\<([^\<]+)\>", type)
        # Extract the inner type and continue processing
        type = matches[0]
```

You'd also need an `import re` up top